docs/tutorial/notebook/02_datasets.ipynb

```diff
@@ -0,0 +1,449 @@
+{
```

Comment:
<p>Standard convention for object detection lidar datasets (e.g. KITTI / Waymo) is x(red) = forward, <em>y (green)</em> = left, <em>z</em> (blue) = <em>up</em>. Add XYZ labels to the axis.</p><p> </p><p>Mention that datasets may instead of r (lidar reflectance) as feature, or no features at all.</p><p> </p><p>A dataset is partitioned into train(ing), valid(ation) and test(ing) splits. Models use data from the training split to update network weights and performance is measured on the validation split. Hyperparameters can be adjusted to optimize performance on the validation split. The test split is never used during the training step and is only used to report the final performance.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>